### **Project Information**

- Project: Video Games Analysis
- Project creator: Kiet Truong
- Sample Data: Video Games Analysis (Source: Kaggle)

### **VIDEO GAMES ANALYSIS**

### **I. Introduction**

In the history of technology, video games have been one of the most promising fields. Game playing is a passion that most people share, whether they are children, teenagers, or adults. Therefore, it can be said that the Games market is always an attractive but equally challenging market. However, not all game publishers can find their suceed. By leveraging factors used to measure video game success such as regional sales, ratings from users and professional organisations, let's discover useful and valuable insights through this analysis.

In [2]:
# Import pandas and sqlite3
import pandas as pd
import sqlite3
df = pd.read_csv('/content/drive/MyDrive/Colab Notebooks/Project_raw_data/Video_Games_Sales/Video_Games_Sales_Rawdata.csv')
cnn = sqlite3.connect('data.db')
df.to_sql('RAW',cnn)
%load_ext sql
%sql sqlite:///data.db

In [3]:
# Raw data
df

ID                            GAME PLATFORM  YEAR         GENRE  \
0        1                       Alter Ego       PC  1985    Simulation   
1        2                         SimCity       PC  1988    Simulation   
2        3                            Doom       PC  1992       Shooter   
3        4          Battle Arena Toshinden       PS  1994      Fighting   
4        5                          Diablo       PC  1996  Role-Playing   
...    ...                             ...      ...   ...           ...   
6860  6861                          XCOM 2       PC  2016      Strategy   
6861  6862                          XCOM 2      PS4  2016      Strategy   
6862  6863                          XCOM 2     XOne  2016      Strategy   
6863  6864  Zero Escape: Zero Time Dilemma      3DS  2016     Adventure   
6864  6865  Zero Escape: Zero Time Dilemma      PSV  2016     Adventure   

                        PUBLISHER  NA_SALES  EU_SALES  JP_SALES  OTHERS_SALES  \
0                      Activision      0.00      0.03      0.00          0.01   
1                           Maxis      0.00      0.02      0.00          0.01   
2                     id Software      0.02      0.00      0.00          0.00   
3     Sony Computer Entertainment      0.39      0.26      0.53          0.08   
4                      Activision      0.01      1.58      0.00          0.00   
...                           ...       ...       ...       ...           ...   
6860         Take-Two Interactive      0.09      0.12      0.00          0.02   
6861         Take-Two Interactive      0.04      0.08      0.00          0.02   
6862         Take-Two Interactive      0.02      0.02      0.00          0.00   
6863                  Aksys Games      0.05      0.00      0.02          0.01   
6864                  Aksys Games      0.03      0.00      0.02          0.01   

      GLOBAL_SALES  CRITIC_SCORE  CRITIC_COUNT  USER_SCORE  USER_COUNT  
0             0.03            59             9         5.8          19  
1             0.03            64            75         2.2        4572  
2             0.03            85            44         8.2        1796  
3             1.27            69             4         6.3           4  
4             1.59            94            12         8.7         850  
...            ...           ...           ...         ...         ...  
6860          0.23            88           104         7.1        1221  
6861          0.14            88            28         8.0         116  
6862          0.05            87            17         8.1          40  
6863          0.08            81            18         8.5          72  
6864          0.06            83            31         8.3         116  

[6865 rows x 15 columns]

Raw Data Explaination:

1. ID: Unique id stand for each video game.
2. GAME: The name of the video game
3. PLATFORM: The platform on which the game was released, such as PlayStation, Xbox, Nintendo, etc.
4. YEAR: The year in which the game was released.
5. GENRE: The genre of the video game, such as action, adventure, sports, etc.
6. PUBLISHER: The company responsible for publishing the game.
7. NA_SALES: The sales (in millions) of the game in North America.
8. EU_SALES: The sales (in millions) of the game in Europe.
9. JP_SALES: The sales (in millions) of the game in Japan.
10. OTHERS_SALES: The sales (in millions) of the game in other regions.
11. GLOBAL_SALES: The total sales (in millions) of the game across the world.
12. CRITIC_SCORE: The average score given to the game by professional critics.
13. CRITIC_COUNT: The number of critics who reviewed the game.
14. USER_SCORE: The average score given to the game by users.
15. USER_COUNT: The number of users who reviewed the game.

### **II. Findings and Analysis**

#### **1. Market Research**

Before we dive deeper, we need to and identify popular games, genres, platforms, and publishers. This can be useful for industry professionals to make informed decisions about game development and marketing strategies.

Note:
- There are some games that are released on different platforms, so that the total global sales of those games will be aggregated.
- There are a lot of publishers so that the project author will just listed top 20 publisher because they contributed to 90% of the total global sales.

In [4]:
# A. Top 10 games based on Global Sales.
%%sql
SELECT
	RANK () OVER (ORDER BY SUM(GLOBAL_SALES) DESC) AS RANK,
	GAME,
	YEAR,
	GENRE,
	ROUND(SUM(GLOBAL_SALES),2) AS TOTAL_GLOBAL_SALES
FROM RAW
GROUP BY GAME
LIMIT 10

 * sqlite:///data.db
Done.


[(1, 'Wii Sports', 2006, 'Sports', 82.53),
 (2, 'Grand Theft Auto V', 2013, 'Action', 56.57),
 (3, 'Mario Kart Wii', 2008, 'Racing', 35.52),
 (4, 'Wii Sports Resort', 2009, 'Sports', 32.77),
 (5, 'Call of Duty: Modern Warfare 3', 2011, 'Shooter', 30.59),
 (6, 'New Super Mario Bros.', 2006, 'Platform', 29.8),
 (7, 'Call of Duty: Black Ops II', 2012, 'Shooter', 29.4),
 (8, 'Call of Duty: Black Ops', 2010, 'Shooter', 29.19),
 (9, 'Wii Play', 2006, 'Misc', 28.92),
 (10, 'New Super Mario Bros. Wii', 2009, 'Platform', 28.32)]

In [5]:
# B. Genres based on Global Sales.
%%sql
SELECT
	RANK () OVER (ORDER BY SUM(GLOBAL_SALES) DESC) AS RANK,
	GENRE,
	COUNT(DISTINCT GAME) AS NO_OF_GAMES,
	ROUND(SUM(GLOBAL_SALES),2) AS TOTAL_GLOBAL_SALES
FROM RAW
GROUP BY GENRE

 * sqlite:///data.db
Done.


[(1, 'Action', 874, 1204.28),
 (2, 'Sports', 511, 834.61),
 (3, 'Shooter', 505, 816.74),
 (4, 'Role-Playing', 583, 500.0),
 (5, 'Racing', 356, 477.03),
 (6, 'Misc', 280, 416.85),
 (7, 'Platform', 270, 377.8),
 (8, 'Fighting', 260, 249.98),
 (9, 'Simulation', 236, 204.22),
 (10, 'Adventure', 202, 81.11),
 (11, 'Puzzle', 107, 78.9),
 (12, 'Strategy', 226, 69.99)]

In [6]:
# C. Platforms baed on Global Sales
%%sql
SELECT
	RANK () OVER (ORDER BY SUM(GLOBAL_SALES) DESC) AS RANK,
	PLATFORM,
	COUNT(DISTINCT GAME) AS NO_OF_GAMES,
	ROUND(SUM(GLOBAL_SALES),2) AS TOTAL_GLOBAL_SALES
FROM RAW
GROUP BY PLATFORM

 * sqlite:///data.db
Done.


[(1, 'PS2', 1138, 945.04),
 (2, 'X360', 858, 851.82),
 (3, 'PS3', 769, 784.24),
 (4, 'Wii', 480, 659.24),
 (5, 'DS', 464, 382.8),
 (6, 'PS4', 248, 244.23),
 (7, 'XB', 566, 212.37),
 (8, 'PS', 153, 209.87),
 (9, 'PC', 677, 188.95),
 (10, 'PSP', 389, 187.77),
 (11, 'GC', 347, 159.32),
 (12, 'GBA', 236, 130.82),
 (13, 'XOne', 164, 130.27),
 (14, '3DS', 155, 123.26),
 (15, 'WiiU', 89, 66.08),
 (16, 'PSV', 118, 30.88),
 (17, 'DC', 14, 4.55)]

In [7]:
# D. Publishers based on Global Sales.
%%sql
SELECT
	RANK () OVER (ORDER BY SUM(GLOBAL_SALES) DESC) AS RANK,
	PUBLISHER,
	COUNT(DISTINCT GAME) AS NO_OF_GAMES,
	ROUND(SUM(GLOBAL_SALES),2) AS TOTAL_GLOBAL_SALES
FROM RAW
GROUP BY PUBLISHER
LIMIT 20

 * sqlite:///data.db
Done.


[(1, 'Electronic Arts', 401, 867.1),
 (2, 'Nintendo', 291, 850.41),
 (3, 'Activision', 219, 535.74),
 (4, 'Sony Computer Entertainment', 290, 388.1),
 (5, 'Take-Two Interactive', 134, 350.17),
 (6, 'Ubisoft', 284, 340.36),
 (7, 'Microsoft Game Studios', 135, 216.49),
 (8, 'THQ', 180, 162.87),
 (9, 'Sega', 191, 150.04),
 (10, 'Konami Digital Entertainment', 176, 138.02),
 (11, 'Namco Bandai Games', 187, 121.5),
 (12, 'Capcom', 135, 113.32),
 (13, 'Warner Bros. Interactive Entertainment', 52, 108.68),
 (14, 'Square Enix', 99, 108.07),
 (15, 'Atari', 134, 82.71),
 (16, 'LucasArts', 31, 71.79),
 (17, 'Bethesda Softworks', 20, 71.38),
 (18, 'Eidos Interactive', 72, 45.01),
 (19, 'Midway Games', 62, 44.29),
 (20, 'Vivendi Games', 65, 42.66)]

In [8]:
# E. Best seller of each publishers based on the global sales.
%%sql
WITH A AS
(SELECT
	PUBLISHER,
	COUNT(DISTINCT GAME) AS NO_OF_GAMES,
	SUM(GLOBAL_SALES) AS GLOBAL_SALES
FROM RAW
GROUP BY PUBLISHER
ORDER BY GLOBAL_SALES DESC
LIMIT 20)

, B AS
(SELECT
	A.PUBLISHER,
	R.GAME,
	R.GENRE,
	R.GLOBAL_SALES,
	R.CRITIC_SCORE,
	R.USER_SCORE,
	RANK () OVER (PARTITION BY A.PUBLISHER ORDER BY R.GLOBAL_SALES DESC, R.CRITIC_SCORE DESC, R.USER_SCORE DESC) AS RANK
FROM RAW AS R
INNER JOIN A
ON R.PUBLISHER = A.PUBLISHER)

SELECT *
FROM B
WHERE RANK = 1

 * sqlite:///data.db
Done.


[('Activision', 'Call of Duty: Modern Warfare 3', 'Shooter', 14.73, 88, 3.4, 1),
 ('Atari', 'Driver 2', 'Action', 4.73, 62, 7.7, 1),
 ('Bethesda Softworks', 'The Elder Scrolls V: Skyrim', 'Role-Playing', 8.79, 96, 8.4, 1),
 ('Capcom', 'Monster Hunter Freedom Unite', 'Role-Playing', 5.48, 81, 8.7, 1),
 ('Eidos Interactive', 'Tomb Raider', 'Action', 4.63, 91, 8.6, 1),
 ('Electronic Arts', 'FIFA 16', 'Sports', 8.57, 82, 4.3, 1),
 ('Konami Digital Entertainment', 'Metal Gear Solid 2: Sons of Liberty', 'Action', 6.05, 96, 8.7, 1),
 ('LucasArts', 'LEGO Star Wars: The Complete Saga', 'Action', 5.64, 80, 8.9, 1),
 ('Microsoft Game Studios', 'Kinect Adventures!', 'Misc', 21.81, 61, 6.3, 1),
 ('Midway Games', 'Mortal Kombat: Deadly Alliance', 'Fighting', 2.49, 79, 8.8, 1),
 ('Namco Bandai Games', 'Namco Museum', 'Misc', 4.24, 79, 7.3, 1),
 ('Nintendo', 'Wii Sports', 'Sports', 82.53, 76, 8.0, 1),
 ('Sega', 'Sonic Rush', 'Platform', 3.12, 82, 8.5, 1),
 ('Sony Computer Entertainment', 'Gran Turismo 3: A-Spec', 'Racing', 14.98, 95, 8.4, 1),
 ('Square Enix', 'Final Fantasy XII', 'Role-Playing', 5.95, 92, 7.6, 1),
 ('THQ', 'Saints Row 2', 'Action', 3.03, 81, 8.1, 1),
 ('Take-Two Interactive', 'Grand Theft Auto V', 'Action', 21.04, 97, 8.2, 1),
 ('Ubisoft', 'Just Dance 3', 'Misc', 10.12, 74, 7.8, 1),
 ('Vivendi Games', 'Half-Life', 'Shooter', 4.12, 96, 9.1, 1),
 ('Warner Bros. Interactive Entertainment', 'Batman: Arkham City', 'Action', 5.48, 96, 8.6, 1)]

#### **2. Competitors Analysis**

By identifying the strengths and weaknesses of competitors in the same field, we can more easily and conveniently reach a specific market or release a new video game.

##### **2.1 Market Share (%)**

In [9]:
# Identify the sales contribution in each region (NA, EU, JP and Others)
%%sql
SELECT
	PUBLISHER,
	ROUND(SUM(GLOBAL_SALES),2) AS GLOBAL_SALES,
	ROUND(SUM(NA_SALES),2) AS NA_SALES,
	ROUND(SUM(EU_SALES),2) AS EU_SALES,
	ROUND(SUM(JP_SALES),2) AS JP_SALES,
	ROUND(SUM(OTHERS_SALES),2) AS OTHERS_SALES,
	ROUND((SUM(NA_SALES)/SUM(GLOBAL_SALES))*100,2) AS "NA_%",
	ROUND((SUM(EU_SALES)/SUM(GLOBAL_SALES))*100,2) AS "EU_%",
	ROUND((SUM(JP_SALES)/SUM(GLOBAL_SALES))*100,2) AS "JP_%",
	ROUND((SUM(OTHERS_SALES)/SUM(GLOBAL_SALES))*100,2) AS "OTHERS_%"
FROM RAW
GROUP BY PUBLISHER
ORDER BY GLOBAL_SALES DESC
LIMIT 20

 * sqlite:///data.db
Done.


[('Electronic Arts', 867.1, 464.48, 288.91, 10.83, 102.79, 53.57, 33.32, 1.25, 11.85),
 ('Nintendo', 850.41, 371.48, 242.64, 174.05, 62.43, 43.68, 28.53, 20.47, 7.34),
 ('Activision', 535.74, 306.82, 166.54, 5.31, 57.01, 57.27, 31.09, 0.99, 10.64),
 ('Sony Computer Entertainment', 388.1, 176.87, 117.53, 39.92, 53.78, 45.57, 30.28, 10.29, 13.86),
 ('Take-Two Interactive', 350.17, 187.82, 106.24, 5.56, 50.55, 53.64, 30.34, 1.59, 14.44),
 ('Ubisoft', 340.36, 175.03, 121.28, 6.41, 37.61, 51.42, 35.63, 1.88, 11.05),
 ('Microsoft Game Studios', 216.49, 138.9, 58.16, 2.83, 16.61, 64.16, 26.86, 1.31, 7.67),
 ('THQ', 162.87, 93.97, 47.25, 2.51, 19.11, 57.7, 29.01, 1.54, 11.73),
 ('Sega', 150.04, 69.99, 50.45, 12.87, 16.59, 46.65, 33.62, 8.58, 11.06),
 ('Konami Digital Entertainment', 138.02, 51.23, 45.34, 21.76, 19.61, 37.12, 32.85, 15.77, 14.21),
 ('Namco Bandai Games', 121.5, 51.64, 33.44, 24.75, 11.66, 42.5, 27.52, 20.37, 9.6),
 ('Capcom', 113.32, 48.97, 28.1, 25.94, 10.15, 43.21, 24.8, 22.89, 8.96),
 ('Warner Bros. Interactive Entertainment', 108.68, 57.96, 36.72, 0.9, 13.19, 53.33, 33.79, 0.83, 12.14),
 ('Square Enix', 108.07, 39.15, 23.97, 34.44, 10.62, 36.23, 22.18, 31.87, 9.83),
 ('Atari', 82.71, 48.69, 18.63, 8.64, 6.6, 58.87, 22.52, 10.45, 7.98),
 ('LucasArts', 71.79, 41.46, 21.13, 0.15, 9.06, 57.75, 29.43, 0.21, 12.62),
 ('Bethesda Softworks', 71.38, 34.74, 26.74, 1.25, 8.65, 48.67, 37.46, 1.75, 12.12),
 ('Eidos Interactive', 45.01, 23.48, 15.59, 1.21, 4.71, 52.17, 34.64, 2.69, 10.46),
 ('Midway Games', 44.29, 27.9, 11.89, 0.12, 4.33, 62.99, 26.85, 0.27, 9.78),
 ('Vivendi Games', 42.66, 28.5, 10.83, 0.23, 3.08, 66.81, 25.39, 0.54, 7.22)]

**Result:**

=> Most of publishers are focus on the NA market, we can see a big gap between NA and the others. Next is the EU market, then Others and final is Japan. This is understandable when Japan is just a country when compared with other regions (including more countries). However, that is not to say that we do not need to pay attention to the Japanese market. There are still some breakthroughs when Nintendo, Namco Bandai Games, Capcom and Square Enix recorded a large amount of revenue coming from the Japanese market. Since most of them only focus on EU and NA region, this could lead to JP being an area that still has a lot of potential and the competitiveness is likely to be much less than the rest.

##### **2.2 Platform Competition**

In [10]:
# Platform research
%%sql
WITH A AS
(SELECT
	PUBLISHER,
	COUNT(DISTINCT GAME) AS TOTAL_GAMES,
	SUM(GLOBAL_SALES) AS GLOBAL_SALES
FROM RAW
GROUP BY PUBLISHER
ORDER BY GLOBAL_SALES DESC
LIMIT 20)

SELECT
	R.PUBLISHER,
	A.TOTAL_GAMES,
	COUNT(DISTINCT CASE WHEN PLATFORM = "PS2" THEN GAME END) AS PS2,
	COUNT(DISTINCT CASE WHEN PLATFORM = "X360" THEN GAME END) AS X360,
	COUNT(DISTINCT CASE WHEN PLATFORM = "PS3" THEN GAME END) AS PS3,
	COUNT(DISTINCT CASE WHEN PLATFORM = "Wii" THEN GAME END) AS Wii,
	COUNT(DISTINCT CASE WHEN PLATFORM = "DS" THEN GAME END) AS DS,
	COUNT(DISTINCT CASE WHEN PLATFORM = "PS4" THEN GAME END) AS PS4,
	COUNT(DISTINCT CASE WHEN PLATFORM = "PS" THEN GAME END) AS PS,
	COUNT(DISTINCT CASE WHEN PLATFORM = "XB" THEN GAME END) AS XB,
	COUNT(DISTINCT CASE WHEN PLATFORM = "PC" THEN GAME END) AS PC,
	COUNT(DISTINCT CASE WHEN PLATFORM = "PSP" THEN GAME END) AS PSP,
	COUNT(DISTINCT CASE WHEN PLATFORM = "GC" THEN GAME END) AS GC,
	COUNT(DISTINCT CASE WHEN PLATFORM = "GBA" THEN GAME END) AS GBA,
	COUNT(DISTINCT CASE WHEN PLATFORM = "XOne" THEN GAME END) AS XOne,
	COUNT(DISTINCT CASE WHEN PLATFORM = "3DS" THEN GAME END) AS "3DS",
	COUNT(DISTINCT CASE WHEN PLATFORM = "WiiU" THEN GAME END) AS WiiU,
	COUNT(DISTINCT CASE WHEN PLATFORM = "PSV" THEN GAME END) AS PSV,
	COUNT(DISTINCT CASE WHEN PLATFORM = "DC" THEN GAME END) AS DC
FROM RAW AS R
INNER JOIN A
ON R.PUBLISHER = A.PUBLISHER
GROUP BY R.PUBLISHER
ORDER BY A.TOTAL_GAMES DESC

 * sqlite:///data.db
Done.


[('Electronic Arts', 401, 149, 134, 113, 62, 42, 26, 18, 107, 107, 50, 76, 22, 27, 4, 2, 3, 0),
 ('Nintendo', 291, 0, 0, 0, 56, 71, 0, 0, 0, 0, 0, 47, 42, 0, 44, 33, 0, 0),
 ('Sony Computer Entertainment', 290, 111, 0, 72, 0, 0, 18, 28, 0, 0, 66, 0, 0, 0, 0, 0, 21, 0),
 ('Ubisoft', 284, 54, 72, 54, 45, 30, 23, 4, 37, 59, 23, 24, 16, 21, 11, 15, 9, 1),
 ('Activision', 219, 63, 80, 66, 41, 26, 18, 15, 49, 45, 14, 30, 18, 14, 5, 7, 1, 0),
 ('Sega', 191, 47, 37, 37, 26, 17, 2, 0, 30, 25, 16, 17, 5, 1, 10, 2, 4, 11),
 ('Namco Bandai Games', 187, 42, 35, 43, 13, 13, 16, 9, 3, 7, 21, 5, 3, 7, 11, 3, 12, 1),
 ('THQ', 180, 53, 49, 36, 25, 25, 0, 1, 24, 22, 17, 28, 24, 0, 2, 1, 0, 0),
 ('Konami Digital Entertainment', 176, 60, 31, 25, 22, 16, 6, 7, 17, 11, 25, 7, 9, 5, 4, 0, 1, 0),
 ('Microsoft Game Studios', 135, 0, 54, 0, 0, 0, 0, 0, 47, 23, 0, 0, 0, 17, 0, 0, 0, 0),
 ('Capcom', 135, 45, 24, 27, 11, 9, 7, 4, 9, 14, 12, 14, 9, 2, 5, 2, 3, 0),
 ('Take-Two Interactive', 134, 36, 56, 44, 15, 6, 14, 5, 27, 38, 12, 2, 2, 14, 0, 1, 1, 0),
 ('Atari', 134, 51, 12, 5, 9, 9, 0, 2, 26, 20, 11, 13, 18, 0, 0, 0, 0, 0),
 ('Square Enix', 99, 17, 18, 17, 4, 25, 8, 0, 0, 17, 15, 0, 1, 6, 5, 0, 3, 0),
 ('Eidos Interactive', 72, 35, 13, 6, 10, 9, 0, 4, 19, 16, 5, 5, 1, 0, 0, 0, 0, 0),
 ('Vivendi Games', 65, 27, 7, 4, 8, 8, 0, 0, 18, 19, 4, 10, 8, 0, 0, 0, 0, 0),
 ('Midway Games', 62, 39, 8, 6, 7, 3, 0, 1, 23, 1, 4, 10, 2, 0, 0, 0, 0, 0),
 ('Warner Bros. Interactive Entertainment', 52, 3, 23, 19, 5, 8, 15, 0, 0, 15, 1, 1, 0, 12, 10, 12, 6, 0),
 ('LucasArts', 31, 14, 8, 6, 7, 7, 0, 0, 5, 10, 7, 4, 1, 0, 1, 0, 0, 0),
 ('Bethesda Softworks', 20, 0, 10, 11, 2, 0, 5, 0, 1, 13, 0, 0, 0, 6, 0, 0, 0, 0)]

**Result:**

- Electronic Arts: PS2, X360, PS3, XB, PC
- Ubisoft: PS3, PS3, X360, XB, PC
- Sony Computer Entertainment: PS2, PS3, PSP, PSV
- Nintendo: Wii, DS, GBA, 3DS
- Activision: PC, PS2, PS3, X360, Wii
- THQ: X360, PS2, PS3
- Sega: PS2, X360, PS3
- Namco Bandai Games: PS3, PS2, X360
- Konami Digital Entertainment: PS2, X360, PS3
- Atari: PS2, XB, PC
- Take-Two Interactive: X360, PS3, PS2
- Microsoft Game Studios: X360, XB, PC
- Capcom: PS2, PS3, X360
- Square Enix: DS, X360, PS3
- Eidos Interactive: PS2, X360
- Midway Games: PS2, XB, PC
- Warner Bros. Interactive Entertainment: X360, PS3, PC
- Disney Interactive Studios: Wii, DS, X360
- LucasArts: PS2, PC
- Bethesda Softworks: X360, PS3, PC

=> Through the results table above, we can see that most publishers choose the Playstation series as the main platform to release games. Next up is the Xbox 360 and PC. So these three platforms are the most popular. There are quite a few publishers choosing other platforms, this may be because other lines have fallen out of vogue and favored by the masses.

However, as one of the first pioneers in the world, Nintendo chose to go against the crowd, mainly releasing games for handheld consoles. This can be considered as one of the reasons for Nintendo's success to this day. One of the best examples of creating your own style and shaping your own products.

With fourth place on the leaderboard, Nintendo shows us that we should not have to follow the crowd to succeed. This may be an approach for new publishers rather than joining the crowd where competition is already evident.

In [11]:
# Statistics of the number of games across each genre
%%sql
WITH A AS (SELECT
	PUBLISHER,
	COUNT(DISTINCT GAME) AS NO_OF_GAMES,
	ROUND(SUM(GLOBAL_SALES),2) AS GLOBAL_SALES
FROM RAW
GROUP BY PUBLISHER
ORDER BY GLOBAL_SALES DESC LIMIT 20)

SELECT
	R.PUBLISHER,
	A.NO_OF_GAMES,
	COUNT(DISTINCT CASE WHEN GENRE = 'Action' THEN GAME END) AS SPORT,
	COUNT(DISTINCT CASE WHEN GENRE = 'Sports' THEN GAME END) AS ACTION,
	COUNT(DISTINCT CASE WHEN GENRE = 'Shooter' THEN GAME END) AS SHOOTER,
	COUNT(DISTINCT CASE WHEN GENRE = 'Role-Playing' THEN GAME END) AS ROLE_PLAYING,
	COUNT(DISTINCT CASE WHEN GENRE = 'Racing' THEN GAME END) AS RACING,
	COUNT(DISTINCT CASE WHEN GENRE = 'Misc' THEN GAME END) AS MICS,
	COUNT(DISTINCT CASE WHEN GENRE = 'Platform' THEN GAME END) AS PLATFORM,
	COUNT(DISTINCT CASE WHEN GENRE = 'Fighting' THEN GAME END) AS FIGHTING,
	COUNT(DISTINCT CASE WHEN GENRE = 'Simulation' THEN GAME END) AS SIMULATION,
	COUNT(DISTINCT CASE WHEN GENRE = 'Adventure' THEN GAME END) AS ADVENTURE,
	COUNT(DISTINCT CASE WHEN GENRE = 'Puzzle' THEN GAME END) AS PUZZLE,
	COUNT(DISTINCT CASE WHEN GENRE = 'Strategy' THEN GAME END) AS STRATEGY
FROM RAW AS R
INNER JOIN A
ON R.PUBLISHER = A.PUBLISHER
GROUP BY R.PUBLISHER
ORDER BY A.GLOBAL_SALES DESC

 * sqlite:///data.db
Done.


[('Electronic Arts', 401, 45, 156, 56, 13, 44, 10, 4, 12, 43, 6, 5, 7),
 ('Nintendo', 291, 38, 23, 11, 50, 15, 40, 41, 6, 14, 17, 24, 12),
 ('Activision', 219, 63, 31, 42, 19, 13, 22, 12, 3, 5, 0, 1, 8),
 ('Sony Computer Entertainment', 290, 53, 50, 34, 22, 36, 26, 32, 9, 5, 16, 3, 4),
 ('Take-Two Interactive', 134, 32, 43, 23, 2, 11, 4, 3, 0, 2, 5, 0, 9),
 ('Ubisoft', 284, 67, 20, 43, 27, 18, 28, 17, 12, 22, 16, 2, 12),
 ('Microsoft Game Studios', 135, 17, 14, 25, 14, 15, 15, 4, 4, 12, 2, 1, 12),
 ('THQ', 180, 42, 5, 15, 5, 31, 10, 18, 22, 3, 6, 2, 21),
 ('Sega', 191, 36, 32, 15, 24, 14, 13, 22, 10, 7, 3, 2, 13),
 ('Konami Digital Entertainment', 176, 46, 25, 14, 11, 2, 11, 20, 7, 25, 10, 3, 2),
 ('Namco Bandai Games', 187, 34, 5, 15, 41, 12, 7, 6, 45, 10, 4, 5, 3),
 ('Capcom', 135, 54, 0, 7, 10, 7, 5, 19, 18, 1, 11, 1, 2),
 ('Warner Bros. Interactive Entertainment', 52, 34, 0, 4, 0, 0, 3, 2, 3, 0, 1, 3, 2),
 ('Square Enix', 99, 14, 0, 6, 65, 0, 2, 0, 2, 1, 2, 3, 4),
 ('Atari', 134, 19, 6, 14, 20, 14, 9, 6, 20, 4, 5, 7, 10),
 ('LucasArts', 31, 11, 0, 9, 1, 2, 0, 0, 0, 3, 1, 0, 4),
 ('Bethesda Softworks', 20, 5, 1, 6, 5, 2, 0, 0, 0, 0, 0, 0, 1),
 ('Eidos Interactive', 72, 24, 2, 18, 4, 5, 2, 1, 3, 2, 2, 3, 6),
 ('Midway Games', 62, 8, 18, 7, 3, 8, 6, 2, 8, 0, 1, 1, 0),
 ('Vivendi Games', 65, 14, 1, 17, 1, 2, 1, 13, 1, 1, 4, 0, 10)]

##### **2.3 Games and Genres**

**Result**

- Electronic Arts: Sports, Shooter, Action, Racing, Simulation
- Ubisoft: Action, Shooter, Misc, Simulation
- Sony Computer Entertainment: Action, Sports, Shooter, Racing
- Nintendo: Role-Playing, Action, Misc, Platform
- Activision: Action, Shooter, Sports
- THQ: Action, Racing, Fighting
- Sega: Action, Sports, Role-Playing
- Namco Bandai Games: Fighting, Role-Playing, Action
- Konami Digital Entertainment: Action, Sports
- Atari: Fighting, Role-Playing, Action
- Take-Two Interactive: Sports, Action, Shooter
- Microsoft Game Studios: Shooter, Action, Strategy
- Capcom: Action, Platform, Fighting
- Square Enix: Role-Playing
- Eidos Interactive: Action, Shooter
- Midway Games: Sports
- Warner Bros. Interactive Entertainment: Action
- Disney Interactive Studios: Action
- LucasArts: Action, Shooter
- Bethesda Softworks: Action, Shooter

=> Through the results table we have, it can be seen that Action, Sports, Shooter, Role-Playing are the 4 genres with the widest coverage. This is understandable because most of the studios' revenue comes from the North America (NA) region and this region has long loved action, shooting and role-playing games. However, there are also some game publishers that make their own mark such as Microsoft with its strategy game series and Konami is sports

It is also worth noting that by mainly releasing only certain genres can bring great benefits to publishers. Because when it comes to that type of game, most users will feel secure about the quality of the game when it is released by that genuine company.

For example:
- Nintendo: Pokemon Series (Role - Playing)
- Microsoft: Age of Empires (Strategy)
- Konami: Efootball (Sports)

This will definitely be an advantage when they release new games as they have had certain achievements in the past with previously released titles.


#### **3. Historical Performance Analysis**

Based on data from I and II. We can identify the main trends of game publishers for the period from 1985 to 2016. However, just looking at the general trend is not convincing enough to strength the argument. To have an objective perspective in assessing the potential and opportunities of game publishers in the future. We need to analyze historical data, and combine it with the Score (Critic or User)

##### **3.1 Critic or User?**

First, we need to check if Critic score or User score will bring more benefits by comparing them with global sales.

In [12]:
# A. Set Tiers for games
%%sql
WITH A AS (SELECT
	ID,
	GENRE,
	PLATFORM,
	GLOBAL_SALES,
	CRITIC_SCORE,
	USER_SCORE,
	CASE
		WHEN CRITIC_SCORE <= 25 THEN 'Tier 4'
		WHEN CRITIC_SCORE > 25 AND CRITIC_SCORE <= 50 THEN 'Tier 3'
		WHEN CRITIC_SCORE > 50 AND CRITIC_SCORE <= 75 THEN 'Tier 2' ELSE 'Tier 1'
	END AS CRITIC_RANK,
		CASE
		WHEN USER_SCORE <= 2.5 THEN 'Tier 4'
		WHEN USER_SCORE > 2.5 AND USER_SCORE <= 5 THEN 'Tier 3'
		WHEN USER_SCORE > 5 AND USER_SCORE <= 7.5 THEN 'Tier 2' ELSE 'Tier 1'
	END AS USER_RANK
FROM RAW)

SELECT * FROM A
LIMIT 10

 * sqlite:///data.db
Done.


[(1, 'Simulation', 'PC', 0.03, 59, 5.8, 'Tier 2', 'Tier 2'),
 (2, 'Simulation', 'PC', 0.03, 64, 2.2, 'Tier 2', 'Tier 4'),
 (3, 'Shooter', 'PC', 0.03, 85, 8.2, 'Tier 1', 'Tier 1'),
 (4, 'Fighting', 'PS', 1.27, 69, 6.3, 'Tier 2', 'Tier 2'),
 (5, 'Role-Playing', 'PC', 1.59, 94, 8.7, 'Tier 1', 'Tier 1'),
 (6, 'Misc', 'PS', 1.92, 92, 7.4, 'Tier 1', 'Tier 2'),
 (7, 'Action', 'PS', 5.05, 91, 9.0, 'Tier 1', 'Tier 1'),
 (8, 'Puzzle', 'PS', 0.14, 83, 7.4, 'Tier 1', 'Tier 2'),
 (9, 'Fighting', 'PS', 5.74, 89, 8.9, 'Tier 1', 'Tier 1'),
 (10, 'Action', 'PS', 4.63, 91, 8.6, 'Tier 1', 'Tier 1')]

Based on Critic Score and User Score data, I divided the Critic Score and User Score into four equal parts.

Critic Score:
- Tier 4: <= 25
- Tier 3: > 25 and <= 50
- Tier 2: > 50 and <= 75
- Tier 1: > 75

User Score:
- Tier 4: <= 2.5
- Tier 3: > 2.5 and <= 5
- Tier 2: > 5 and <= 7.5
- Tier 1: > 7.5

In [13]:
# B. Define global sales of Tier 1 games
%%sql
WITH A AS (SELECT
	ID,
	GENRE,
	PLATFORM,
	GLOBAL_SALES,
	CRITIC_SCORE,
	USER_SCORE,
	CASE
		WHEN CRITIC_SCORE <= 25 THEN 'Tier 4'
		WHEN CRITIC_SCORE > 25 AND CRITIC_SCORE <= 50 THEN 'Tier 3'
		WHEN CRITIC_SCORE > 50 AND CRITIC_SCORE <= 75 THEN 'Tier 2' ELSE 'Tier 1'
	END AS CRITIC_RANK,
		CASE
		WHEN USER_SCORE <= 2.5 THEN 'Tier 4'
		WHEN USER_SCORE > 2.5 AND USER_SCORE <= 5 THEN 'Tier 3'
		WHEN USER_SCORE > 5 AND USER_SCORE <= 7.5 THEN 'Tier 2' ELSE 'Tier 1'
	END AS USER_RANK
FROM RAW)

SELECT
	ROUND(SUM(CASE WHEN CRITIC_RANK = 'Tier 1' THEN GLOBAL_SALES END),2) AS CRITIC_TIER1_GLOBALSALES,
	ROUND(SUM(CASE WHEN USER_RANK = 'Tier 1' THEN GLOBAL_SALES END),2) AS USER_TIER1_GLOBALSALES
FROM A

 * sqlite:///data.db
Done.


[(3587.75, 3205.62)]

Through the results above, we can see that Critic Score is more legit than the User Score. The value that the Tier 1 games based on Critic Score is better than the Tier 1 games based on User Score. So that, from now on, we will use Critic Score as a mainstream and User Score for other purposes (if any).

##### **3.2 Discover Tier 1 Games**

So, we all know the importance from Tier 1 games to the revenue. Next, we need to know that a game will have to reach how many scores to achieve Tier 1 in each Genre and Platform.

In [14]:
# A. Define Tier 1 minimum score
%%sql
WITH A AS (SELECT
	ID,
	GENRE,
	PLATFORM,
	GLOBAL_SALES,
	CRITIC_SCORE,
	CASE
		WHEN CRITIC_SCORE <= 25 THEN 'Tier 4'
		WHEN CRITIC_SCORE > 25 AND CRITIC_SCORE <= 50 THEN 'Tier 3'
		WHEN CRITIC_SCORE > 50 AND CRITIC_SCORE <= 75 THEN 'Tier 2' ELSE 'Tier 1'
	END AS CRITIC_RANK
FROM RAW)

,B AS (SELECT
	GENRE,
	PLATFORM,
	ID,
	CRITIC_SCORE,
	CRITIC_RANK,
	ROW_NUMBER () OVER (PARTITION BY GENRE, PLATFORM ORDER BY CRITIC_SCORE) AS ROW_TAKEN
FROM A
WHERE CRITIC_RANK = 'Tier 1')

SELECT
	GENRE,
	PLATFORM,
	CRITIC_SCORE AS MIN_SCORE
FROM B
WHERE ROW_TAKEN = 1
LIMIT 16

 * sqlite:///data.db
Done.


[('Action', '3DS', 77),
 ('Action', 'DS', 76),
 ('Action', 'GBA', 76),
 ('Action', 'GC', 76),
 ('Action', 'PC', 76),
 ('Action', 'PS', 77),
 ('Action', 'PS2', 76),
 ('Action', 'PS3', 76),
 ('Action', 'PS4', 76),
 ('Action', 'PSP', 76),
 ('Action', 'PSV', 76),
 ('Action', 'Wii', 76),
 ('Action', 'WiiU', 76),
 ('Action', 'X360', 76),
 ('Action', 'XB', 76),
 ('Action', 'XOne', 76)]

Explaination:
Let's take Action Genre as an example. There are a lot of action games have been released on different platforms. If the next game they release is Action and on PS4, it must score at least 76 in Critic to make it to the Tier 1 game list.

##### **3.3 Historical Performance Analysis**

Based on the findings above, we have deep dive into the video games analysis by using Genres and Platforms. More specifically, thanks to the set of tiers for the games, we can clearly see that the sales are heavily dependent on the scores from Critic. Games in the Tier 1 list (games with scores higher than 75) accounted for 67.5% of the total revenue. So it's very helpful for game publishers and marketing teams to know what genres and platforms they've been doing great.

In [15]:
# Define the succeed chance
%%sql
WITH A AS (SELECT
	ID,
	GENRE,
	PLATFORM,
	GLOBAL_SALES,
	CRITIC_SCORE,
	CASE
		WHEN CRITIC_SCORE <= 25 THEN 'Tier 4'
		WHEN CRITIC_SCORE > 25 AND CRITIC_SCORE <= 50 THEN 'Tier 3'
		WHEN CRITIC_SCORE > 50 AND CRITIC_SCORE <= 75 THEN 'Tier 2' ELSE 'Tier 1'
	END AS CRITIC_RANK
FROM RAW)

,B AS (SELECT
	GENRE,
	PLATFORM,
	ID,
	CRITIC_SCORE,
	CRITIC_RANK,
	ROW_NUMBER () OVER (PARTITION BY GENRE, PLATFORM ORDER BY CRITIC_SCORE) AS ROW_TAKEN
FROM A
WHERE CRITIC_RANK = 'Tier 1')

, C AS (SELECT
	GENRE,
	PLATFORM,
	CRITIC_SCORE AS MIN_SCORE
FROM B
WHERE ROW_TAKEN = 1)

, D AS (SELECT
	R.ID,
	R.PLATFORM,
	R.GENRE,
	R.CRITIC_SCORE,
	R.PUBLISHER,
	C.MIN_SCORE
FROM RAW AS R
INNER JOIN C
ON R.GENRE = C.GENRE AND R.PLATFORM = C.PLATFORM)

SELECT
	GENRE,
	PLATFORM,
	COUNT(DISTINCT ID) AS NO_OF_GAMES,
	COUNT(CASE WHEN CRITIC_SCORE >= MIN_SCORE THEN ID END) AS TIER1_GAME,
	ROUND((CAST(COUNT(CASE WHEN CRITIC_SCORE >= MIN_SCORE THEN ID END) AS FLOAT)/CAST(COUNT(DISTINCT ID) AS FLOAT))*100,2) AS "SUCCEED_CHANCE(%)"
FROM D
WHERE PUBLISHER = 'Electronic Arts'
GROUP BY GENRE, PLATFORM
ORDER BY "SUCCEED_CHANCE(%)" DESC

 * sqlite:///data.db
Done.


[('Adventure', 'Wii', 2, 2, 100.0),
 ('Fighting', 'PS', 1, 1, 100.0),
 ('Misc', 'PC', 1, 1, 100.0),
 ('Platform', 'PS3', 1, 1, 100.0),
 ('Platform', 'X360', 1, 1, 100.0),
 ('Puzzle', 'DS', 1, 1, 100.0),
 ('Puzzle', 'PS2', 1, 1, 100.0),
 ('Racing', 'PS4', 1, 1, 100.0),
 ('Role-Playing', 'PS4', 1, 1, 100.0),
 ('Role-Playing', 'X360', 4, 4, 100.0),
 ('Role-Playing', 'XOne', 1, 1, 100.0),
 ('Shooter', 'PS', 2, 2, 100.0),
 ('Simulation', 'PS3', 1, 1, 100.0),
 ('Simulation', 'X360', 1, 1, 100.0),
 ('Sports', 'PS', 7, 7, 100.0),
 ('Strategy', 'X360', 1, 1, 100.0),
 ('Shooter', 'PC', 22, 20, 90.91),
 ('Sports', 'GC', 34, 30, 88.24),
 ('Shooter', 'XOne', 7, 6, 85.71),
 ('Fighting', 'PS2', 6, 5, 83.33),
 ('Sports', 'XB', 54, 44, 81.48),
 ('Fighting', 'PS3', 5, 4, 80.0),
 ('Fighting', 'X360', 5, 4, 80.0),
 ('Racing', 'PC', 10, 8, 80.0),
 ('Role-Playing', 'PS3', 5, 4, 80.0),
 ('Sports', 'PS3', 50, 39, 78.0),
 ('Role-Playing', 'PC', 9, 7, 77.78),
 ('Fighting', 'XB', 4, 3, 75.0),
 ('Strategy', 'PC', 4, 3, 75.0),
 ('Sports', 'PS2', 73, 54, 73.97),
 ('Sports', 'X360', 60, 44, 73.33),
 ('Sports', 'PC', 11, 8, 72.73),
 ('Fighting', 'GC', 3, 2, 66.67),
 ('Puzzle', 'Wii', 3, 2, 66.67),
 ('Shooter', 'PS4', 6, 4, 66.67),
 ('Racing', 'PS2', 22, 14, 63.64),
 ('Racing', 'X360', 11, 7, 63.64),
 ('Sports', 'PS4', 16, 10, 62.5),
 ('Sports', 'XOne', 16, 10, 62.5),
 ('Racing', 'XB', 13, 8, 61.54),
 ('Shooter', 'GC', 10, 6, 60.0),
 ('Shooter', 'X360', 20, 12, 60.0),
 ('Shooter', 'PS3', 14, 8, 57.14),
 ('Simulation', 'PC', 31, 17, 54.84),
 ('Shooter', 'XB', 15, 8, 53.33),
 ('Adventure', 'XB', 2, 1, 50.0),
 ('Platform', 'PC', 2, 1, 50.0),
 ('Racing', 'GBA', 2, 1, 50.0),
 ('Role-Playing', 'PS2', 2, 1, 50.0),
 ('Simulation', 'XB', 4, 2, 50.0),
 ('Sports', 'PSV', 2, 1, 50.0),
 ('Sports', 'Wii', 24, 12, 50.0),
 ('Shooter', 'PS2', 17, 8, 47.06),
 ('Racing', 'PS3', 9, 4, 44.44),
 ('Racing', 'PS', 5, 2, 40.0),
 ('Simulation', 'GC', 5, 2, 40.0),
 ('Action', 'PC', 16, 6, 37.5),
 ('Action', 'PS3', 24, 9, 37.5),
 ('Sports', 'PSP', 25, 9, 36.0),
 ('Action', 'X360', 26, 9, 34.62),
 ('Action', 'GBA', 6, 2, 33.33),
 ('Action', 'XB', 12, 4, 33.33),
 ('Racing', 'GC', 9, 3, 33.33),
 ('Shooter', 'GBA', 3, 1, 33.33),
 ('Simulation', 'PS2', 6, 2, 33.33),
 ('Strategy', 'PS2', 3, 1, 33.33),
 ('Action', 'GC', 11, 3, 27.27),
 ('Misc', 'X360', 4, 1, 25.0),
 ('Sports', 'GBA', 8, 2, 25.0),
 ('Racing', 'PSP', 9, 2, 22.22),
 ('Shooter', 'Wii', 5, 1, 20.0),
 ('Sports', 'DS', 10, 2, 20.0),
 ('Action', 'PSP', 7, 1, 14.29),
 ('Simulation', 'Wii', 7, 1, 14.29),
 ('Action', 'PS2', 16, 2, 12.5),
 ('Action', '3DS', 2, 0, 0.0),
 ('Action', 'DS', 8, 0, 0.0),
 ('Action', 'PS', 2, 0, 0.0),
 ('Action', 'PS4', 1, 0, 0.0),
 ('Action', 'PSV', 1, 0, 0.0),
 ('Action', 'Wii', 8, 0, 0.0),
 ('Action', 'WiiU', 1, 0, 0.0),
 ('Action', 'XOne', 1, 0, 0.0),
 ('Adventure', 'PC', 1, 0, 0.0),
 ('Adventure', 'PS2', 1, 0, 0.0),
 ('Adventure', 'PS3', 1, 0, 0.0),
 ('Adventure', 'X360', 1, 0, 0.0),
 ('Fighting', 'DS', 1, 0, 0.0),
 ('Fighting', 'PSP', 2, 0, 0.0),
 ('Fighting', 'Wii', 1, 0, 0.0),
 ('Misc', 'DS', 1, 0, 0.0),
 ('Misc', 'GC', 1, 0, 0.0),
 ('Misc', 'PS3', 3, 0, 0.0),
 ('Misc', 'Wii', 5, 0, 0.0),
 ('Platform', 'GC', 2, 0, 0.0),
 ('Platform', 'PS2', 2, 0, 0.0),
 ('Platform', 'PS4', 1, 0, 0.0),
 ('Platform', 'XB', 2, 0, 0.0),
 ('Platform', 'XOne', 1, 0, 0.0),
 ('Racing', 'DS', 8, 0, 0.0),
 ('Racing', 'Wii', 7, 0, 0.0),
 ('Racing', 'XOne', 1, 0, 0.0),
 ('Role-Playing', 'GBA', 1, 0, 0.0),
 ('Role-Playing', 'GC', 1, 0, 0.0),
 ('Role-Playing', 'PS', 1, 0, 0.0),
 ('Shooter', 'DS', 1, 0, 0.0),
 ('Shooter', 'PSP', 4, 0, 0.0),
 ('Simulation', '3DS', 1, 0, 0.0),
 ('Simulation', 'DS', 12, 0, 0.0),
 ('Simulation', 'GBA', 2, 0, 0.0),
 ('Sports', '3DS', 1, 0, 0.0),
 ('Sports', 'WiiU', 1, 0, 0.0),
 ('Strategy', 'XB', 1, 0, 0.0)]

Method: By dividing the number of tier 1 games by the total number of games released (in each specific genre and platform).

Take Electronic Arts as an example. Through the method above, we can see that they have been very successful with the Sport and Shooter series on the Playstation, Adventure on the Wii or Role-playing on the X360, etc.
Since the games they have released in the above genres are 100% rated, it means that all the games in the above genre are Tier 1.
This can help us to infer a conclusion that is Electronic Arts has always made a big splash when it comes to launching games of this genre.

On the other hand, Electronic Arts has also had failures in the past when they did not have a game that reached Tier 1.
Like the Simulation genre on the DS series, they have released 12 games but none of them reached the top rating.
Besides, Action series on PS2 were not a good experienced for users when they have released 16 games but only 2 games entered Tier 1 and only achieved 12.5% success rate.

However, they also have a growing series of games that have seen a large number of games released.
Like the Sport series on PS3, reaching 74% with 54 out of 73 games reaching tier 1.
Besides showing us the strengths and weaknesses, based on historical result, we can estimate that the next game of this genre on the playstation platform will have at least 74% in success rate.

By looking at the Succeed Chance index, we can generalize the strengths and weaknesses of game publishers.
This can be noted as a benchmark, based on historical data to assess the chances of success of future game releases.



### **III. Conclusion**

Video games have always been one of the entertainment fields with great potential. Nowadays, with the rise of esports, gaming is now not only considered a tool for entertainment but it is also considered a sport.

With the development of technology and the growth of esports, the possibilities for gaming have become limitless. Ultimately, the conclusion is that video games are an ever-evolving industry that requires creativity, dedication and a keen understanding of the gaming market. Publishers must always be on the lookout for new ideas and strategies to stay ahead of the competition. By embracing new technologies and trends, as well as continuing to develop quality titles, Publishers can continue to remain successful in the video games industry. Therefore, it is essential for Publishers to understand the market and stay ahead of the competition by consistently delivering high-quality games.

### **IV. Further Assignments**

Because of some limitations to the data, there are a number of analytical methods that have not been implemented at present. Therefore, the following methods will be considered in the near future when the level of completeness and reliability of the current project proves its success.

- Consumer Insight: We track and analyze our customer data to gain insights into consumer preferences and buying habits, which can be used to tailor marketing strategies and improve customer satisfaction. Through this approach, we can strive to make sure every game we offer is tailored to our customers’ needs and desires.

- Gaming Trends Over Time: This data can be used to better understand the gaming industry and identify trends over time. It can also be used to track the evolution of the gaming market, making it easier to anticipate the next big thing and stay ahead of the competition. With this data, game developers can create more engaging and innovative titles that appeal to their target audience.